install-ghostery.sh

```diff
@@ -7,11 +7,23 @@
 # Before running it, it is recommended to backup the profiles
 # in "~/.ghostery browser".
 #
-# Note: 
+# Note:
 #
 set -Eeuo pipefail
 
-RELEASE_URL="https://get.ghosterybrowser.com/download/linux/en-US"
+case "$1" in
```

Comment:
```suggestion
case "${1:-}" in
```

If you run the script without parameters, $1 is unbound and would fail since it runs in `set -ue`.

Reply:
Ho, you are right. I must admit I stop testing this case once I added the argument. I just force-pushed a new version, which fix the problem.